frontend/components/modal/qr-code/ModalQRCode.vue

```diff
@@ -133,7 +133,7 @@ const props = defineProps<{
 
 const aboveMediumBP = useBreakpoint("md");
 
-const { linkURL } = useLinkURL(props);
+const { linkUrl } = useLinkURL(props);
```

Comment:
Ya I didn't want to change it in the function as it just seemed weird to me. In the last sync we chatted and there will be an issue to do a custom conversion for URL so that we can maintain the uppercase. For now I wanted to do a direct conversion so that we don't need to worry about anything else.

I'll make that issue now though 😊

Reply:
#1007 has been made for this. Maybe you'd want to take a look at it once we're done here?